RideShare/src/app/models/states/location-state.model.ts

```diff
@@ -10,12 +10,14 @@ import { HouseLocation } from '../houselocation.model';
  */
 export interface ILocationState {
   /**
-   * allTrainingLocations: Array<TrainingLocation> - contains all of the training locations known in the database
+   * allTrainingLocations: Array<TrainingLocation> 
```

Comment:
remove commented out value

Reply:
Part of documentation comments, shouldn't remove.